beacon-chain/cache/depositsnapshot/deposit_tree_snapshot.go

```diff
@@ -27,29 +19,27 @@ type DepositTreeSnapshot struct {
 func (ds *DepositTreeSnapshot) CalculateRoot() ([32]byte, error) {
 	size := ds.depositCount
 	index := len(ds.finalized)
-	root := Zerohashes[0]
+	root := trie.ZeroHashes[0]
 	for i := 0; i < DepositContractDepth; i++ {
 		if (size & 1) == 1 {
 			if index == 0 {
-				return [32]byte{}, ErrZeroIndex
+				break
```

Comment:
just wanted to confirm the intended logic here to break out of the for loop but still run the hash.Hash on the return on the bottom.

Reply:
I think this is intended, we want to exit the loop. Will let @saolyn confirm, since she introduced the change